src/main/java/edu/ucla/library/iiif/fester/verticles/S3BucketVerticle.java

```diff
@@ -84,13 +84,14 @@ public void start() throws Exception {
         getJsonConsumer().handler(message -> {
             final JsonObject messageBody = message.body();
             final String action = message.headers().get(Constants.ACTION);
+            final String apiVersion = message.headers().get(Constants.IIIF_API_VERSION);
```

Comment:
This is just in prep for when we treat these differently? Just logging the difference now?

Reply:
I just thought it was something good to log